praw/models/reddit/subreddit.py

```diff
@@ -1751,6 +1757,26 @@ def clear(self):
         """
         self._clear(is_link=False)
 
+    def order(self, flair_template_ids):
+        """Reorder user flair templates.
+
+        :param flair_template_ids: The list of template ids, ordered from top to bottom,
+            where the first item in the list will represent the top-most flair template.
+
+        .. warning:: Omitting a flair template ID will result in its removal.
+
+        For example:
+
+        .. code-block:: python
+
+           subreddit = reddit.subreddit("NAME")
+           template_ids = [t['id'] for t in subreddit.flair.templates]
+           new_order = template_ids[::-1] # creates a new reversed list
+           subreddit.flair.templates.order(new_order)
+
+        """
```

Comment:
For ease of use, it would be great if this method accepted a list of IDs or a list of template objects. That way, PRAW users can write simpler code like:
```python
subreddit = reddit.subreddit("NAME")
new_order = list(subreddit.flair.templates)[::-1] # creates a new reversed list
subreddit.flair.templates.order(new_order)
```
Inside each of the methods, you could check if each list element is a dict and get the ID out of the dict if so.

Reply:
There is no current template object.

Reply:
I had a previous pull request for making such an object but the idea has been scrapped.

Reply:
Aren't flair templates represented as dictionary objects?

```
In [7]: type(list(subreddit.flair.templates)[0])                                
Out[7]: dict
```

Reply:
Oh I see, you want the code itself to extract from dictionary objects. I thought you were asking for a specific Flair class object. That should be manageable, but now we run into errors when using the dictionaries provided by `Submission.flair.choices`.